### PR TITLE
feat: add install script

### PR DESCRIPTION
**Pull request by wuwentao**
# PR Description
add bash script to install midea_ac_lan

## Related issue

fix #X

### CODE REVIEW

File: scripts/install.sh [hunk 1]
```diff
@@ -0,0 +1,130 @@
+#!/bin/bash
+#
+# origin https://github.com/al-one/hass-xiaomi-miot/blob/master/install.sh
+#
+# wget -q -O - https://raw.githubusercontent.com/wuwentao/midea_ac_lan/master/scripts/install.sh | bash -
+# wget -q -O - https://raw.githubusercontent.com/wuwentao/midea_ac_lan/master/scripts/install.sh | ARCHIVE_TAG=v0.4.2 bash -
```
Comment (chemelli74) [inline scripts/install.sh:6]: I would first download `manifest.json` and then take the version from there to avoid more maintenance.

**Review by rokam — APPROVED** [review]
Looks good to me. Although I don't like to use install scripts.

**Review by chemelli74 — CHANGES_REQUESTED** [review]
I would like to avoid the hardcoded version

**Comment by wuwentao** [comment]
@chemelli74  what's the hardcode version info? could you help to confirm ?

there is only a command example with version tag for help, it can be used for version rollback.

**Comment by chemelli74** [comment]
> @chemelli74 what's the hardcode version info? could you help to confirm ?
> 
> there is only a command example with version tag for help, it can be used for version rollback.

Sorry misread from mobile.
Please add a clear info on top is an example.

**Pull request merged**